modulefiles/wflow_macos.lua

```diff
@@ -14,13 +14,12 @@ load(rocoto)
 --]]
 
 if mode() == "load" then
-   LmodMsgRaw([===[Verify the Python virtual environment path \$VENV shown below is correct, "
-set to the correct path otherwise: "
-VENV=$env(VENV) "
-Please do the following to activate python virtual environment:
-       > source \$VENV/bin/activate "
+   LmodMsgRaw([===[Please do the following to activate python virtual environment:
+       > conda activate $VENV "
 ]===])
 end
-if mode() == "unload" then
+
+--[[if mode() == "unload" then
    execute{cmd="deactivate", modeA={"unload"}}
 end
+--]]
```

Comment:
Rather than commenting out these lines, would it be better to remove them?

Reply:
Sure. Will update.